cmake/FindMKL.cmake

```diff
@@ -94,6 +94,7 @@ if (ENABLE_MKLCPU_BACKEND OR ENABLE_MKLGPU_BACKEND)
     list(APPEND MKL_LINK_C ${TBB_LINK})
   endif()
   if(USE_DPCPP_API)
+    find_package(OpenCL REQUIRED)
```

Comment:
Does this also set the library name in some variable, so that line 81 and 85 above aren't necessary?

Reply:
It defines OpenCL::OpenCL target, but I'm not sure if it includes only -lOpenCL or any other extra flags and libraires that we don't want to include.
As the next step of improvement I think we need to integrate MKLConfig.cmake from Intel oneMKL instead of this local FindMKL.cmake.

Reply:
Thanks, the approach in this PR is fine, I was just curious.